src/main/java/de/tum/in/www1/artemis/domain/User.java

```diff
@@ -116,6 +116,13 @@
     @BatchSize(size = 20)
     private Set<Authority> authorities = new HashSet<>();
 
+    @ManyToMany
+    @JoinTable(name = "user_organization", joinColumns = { @JoinColumn(name = "user_id", referencedColumnName = "id") }, inverseJoinColumns = {
+            @JoinColumn(name = "organization_id", referencedColumnName = "id") })
+    @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+    @BatchSize(size = 20)
```

Comment:
remove BatchSize and add JsonIgnoreProperties

Reply:
Done :+1: